src/CMakeLists.txt

```diff
@@ -180,6 +182,7 @@ SET( brewtarget_UIS
     ${UIDIR}/mashStepEditor.ui
     ${UIDIR}/miscEditor.ui
     ${UIDIR}/fermentableEditor.ui
+    ${UIDIR}/equipmentEditor.ui
```

Comment:
Didn't notice it before. I can revert the changes.
Why have you killed it?
When I saw it, I just thought the opposite! haha
"The guys are moving from programatically created ui to .ui files."